anyway/backend_constants.py

```diff
@@ -301,4 +302,25 @@ class InvolvedType(Enum):
     # this is defined based on https://docs.google.com/spreadsheets/d/1qaVV7NKXVYNmnxKZ4he2MKZDAjWPHiHfq-U5dcNZM5k/edit#gid=266079360
     DRIVER = 1
     INJURED_DRIVER = 2
-    INJURED = 3
\ No newline at end of file
+    INJURED = 3
+
+
+class NewsflashLocationQualification(LabeledCode):
+    VERIFIED = 1
+    REJECTED = 2
+    MANUAL = 3
```

Comment:
Hi Shai, We also need a value for news-flashes that did not get any human verification. I wrote it as "none", but maybe "not_verified" may be better. This is actually the status of all current news-flashes.

Reply:
done